db/migrations/20201130231154-create-user-book.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
       },
       status: {
         allowNull: false,
-        type: Sequelize.BOOLEAN
+        type: Sequelize.STRING(10),
+        unique: true
```

Comment:
`unique: true` is going to make it so you cannot have the same status in **any other row in this user book table**. I don't _think_ this is what you want to do, but help me understand if I'm wrong!